v2.9.2

JohnFawkes opened the pull request:
Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

JohnFawkes committed:
v2.9.1

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

JohnFawkes committed:
bug fix

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

thehappydinoa commented on README.md:
```diff
@@ -60,6 +60,12 @@ As a Mi A1 user, I was stuck with Stock Oreo 8.0.0 without Oreo custom ROMs when
 
 ## ⭐ Changelog
```
Can you remove mention of Flip to Shhh from README.md?

thehappydinoa commented on pix3lify.sh:
```diff
@@ -345,17 +337,17 @@ fi
-  echo "==========================================" >> $LOG 2>&1
-  grep "$MODID" -B 1 $DPF >> $LOG 2>&1
+  if grep -qF "com.google.android.dialer" $LIST; then
+    log_print " Collecting Logs for Patches "
```
Can we clarify message?

JohnFawkes replied:
can do

thehappydinoa commented on common/service.sh:
```diff
@@ -2,8 +2,8 @@
 if [ $API -ge 28 ] && [ $FULL ]; then
-  pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"
   am force-stop "com.google.android.apps.wellbeing"
+  pm enable "com.google.android.apps.wellbeing"
```
Why are we doing this?

JohnFawkes replied:
i just removed the autodnd line

pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"

thehappydinoa replied:
But the whole thing is enabled already

thehappydinoa commented on common/uninstall.sh:
```diff
@@ -18,7 +18,7 @@ fi
 ui_print "   Disabling Google's Flip to Shhh..."
 # Disabling Google's Flip to Shhh
-WELLBEING_PREF_FILE=$(find /data/data/com.google.android.apps.wellbeing*/shared_prefs -name "PhenotypePrefs.xml")
+WELLBEING_PREF_FILE=$(find /data/data/com.google.android.apps.wellbeing* -name "PhenotypePrefs.xml")
```
Thought we were removing this

JohnFawkes replied:
removing what? flip to shhh? i didnt know

thehappydinoa replied:
Same as AutoDND

thehappydinoa reviewed (dismissed):
Overall pretty solid, just some traces of AutoDND

JohnFawkes committed:
clarify "patches"

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

JohnFawkes committed:
Remove autoDND/flip to shhh

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

thehappydinoa commented on README.md:
```diff
@@ -56,10 +56,18 @@ As a Mi A1 user, I was stuck with Stock Oreo 8.0.0 without Oreo custom ROMs when
 
 ## ⭐ Features Under Development
```
Should we remove this header?

JohnFawkes committed:
unity update/bug fixes

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

JohnFawkes committed:
add font option

Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

lzlrd committed:
Fonts.xml integration and possible Flip to Shhh

By using @AuroraWright 's port for Flip to Shh and @skittles9823 's
dynamic fonts.xml modification we set Product Sans as a default.

lzlrd commented:
Forgot to add I fixed the Markup issue for lower than Pie devices.

lzlrd commented:
#78 solved
#80 nearly there
#26 rip
#61 cannot be done universally

lzlrd committed:
new dynamic fonts editor by @skittles9823 and digital wellbeing from Q DP1

lzlrd commented:
@Stallix fixed up what you asked in TG. @JohnFawkes is there anything left or shall we push to master?

lzlrd committed:
improve font handling and replace samsung clock font on lockscreen

lzlrd committed:
Update module.prop

thehappydinoa commented on README.md:
```diff
@@ -54,11 +55,18 @@ As a Mi A1 user, I was stuck with Stock Oreo 8.0.0 without Oreo custom ROMs when
+## ⭐ Changelog
 
--   Flip to Shhh (mileage may vary)
+### v2.9.1
```
Bump

lzlrd replied:
Version change, right?

thehappydinoa commented on common/service.sh:
```diff
@@ -2,7 +2,6 @@
 # More info in the main Magisk thread
 
 if [ $API -ge 28 ] && [ $FULL ]; then
-  pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"
```
Figure out how to remove updates before installing WellBeing

lzlrd replied:
Are you referring to Aurora's method? Pretty sure @Stallix tested and it doesn't work.

I guess this line is unnecessary code then too.

If we did want to remove updates it would be a part of the pm --uninstall command or wiping /data/data/x and /data/app/x.

thehappydinoa reviewed (approved):
Check file permissions of `system/app/MarkupGoogle/MarkupGoogle1.apk`

lzlrd commented:
> 
> 
> Check file permissions of `system/app/MarkupGoogle/MarkupGoogle1.apk`

I'm not actually sure how to change that on Windows, would you be able to push a fix?

EDIT: Sorted

lzlrd committed:
Update README.md

lzlrd committed:
Update install.sh

lzlrd committed:
Update service.sh

lzlrd commented:
@thehappydinoa I think I sorted out all the issues there.

lzlrd committed:
Possible perms improvement? Also fix MODPATH -> INSTALLER

This will enforce all app folders and *.apks have correct perms.

lzlrd commented:
@JohnFawkes can u checkout the last commit I did there? Seems it can be improved.

lzlrd committed:
possible improvement

lzlrd committed:
Update install.sh

Lol me dumb

lzlrd commented:
Yay time
